.pre-commit-config.yaml

```diff
@@ -35,6 +35,7 @@ repos:
     rev: 21.8b0
     hooks:
     - id: black
+    - id: black-jupyter
```

Comment:
`black-jupyter` will run on Python and `pyi` files as well, you probably only need that one

Reply:
That's awesome. :)